dev/ory/oathkeeper.yml

```diff
@@ -13,6 +13,15 @@ authenticators:
     config:
       subject: anon
 
+  # cookie_session:
```

Comment:
Shouldn't keep this in here - if its prep for a next step then at least add a comment to indicate when it will become relevant.